app/scripts/modules/core/delivery/manualExecution/manualPipelineExecution.controller.js

```diff
@@ -115,7 +115,12 @@ module.exports = angular.module('spinnaker.core.delivery.manualPipelineExecution
       addTriggers();
       this.triggerUpdated();
 
-      this.showRebakeOption = pipeline.stages.some((stage) => stage.type === 'bake');
+      let bakeStage = pipeline.stages.find(s => s.type === 'bake');
```

Comment:
What is the behavior if the pipeline has more than one bake stage, and the rebake setting is not the same for all?